docs/security.md

```diff
@@ -1,15 +1,17 @@
 Security
 ==========
 
-Applications built on Apex run as native YARN applications on Hadoop. The security framework and apparatus in Hadoop apply to the applications. The default security mechanism in Hadoop is Kerberos.
+Applications built on Apex run as native YARN applications on Hadoop. The security framework and apparatus in Hadoop apply to the applications. Both authentication and SSL aspects of the security framework are covered here.
 
```

Comment:
@PramodSSImmaneni you are right but my point is this page now covers not only (Kerberos) authentication but also SSL as a security aspect. Are you suggesting changing the wording to include "Web services SSL" or something to that effect?